src/routes/docs/tutorials/pagination/step-4/+page.markdoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: tutorial
+title: Add database
+description: Add a database to your React application using Appwrite Web SDK.
+step: 4
+---
+Databases are essential for dynamic applications, ensuring data is organized and accessible. They form the backbone for data storage and management. In this step, we'll integrate a database into our application using the Appwrite Web SDK, setting the stage for more advanced functionalities.
+
+# Create a database {% #create-a-database %}
+In Appwrite's console, navigate to the **Databases** page and click **Create database** and name it "development".
+
+# Create a collection {% #create-collection %}
+To create a collection, head to the **Databases** page, find to your database, and click **Create collection**. Name the collection "Todos" as it will store our todo items.
+
+When setting up your collection, make sure it has these attributes:
+
+{% table %}
+* &nbsp; {% width=10 %}
+* Field {% width=10 %}
+* Type {% width=10 %}
+* Required {% width=10 %}
+* Size {% width=10 %}
+* Min {% width=10 %}
+* Max {% width=10 %}
+---
+* &nbsp;
+* todoId
```

Comment:
Yes, and I thought it would be better as `todoId` than as `taskId`. I see that in the `createDocument()` function I would need to make a change there too.